molecule/default/tasks/ansibleinstancegroup_test.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Create the tower.ansible.com/v1alpha1.AnsibleInstanceGroup
```

Comment:
Same, we can remove this, covered in CI.

Reply:
Maybe we could look in to how to run molecule tests locally.  This may involve breaking up the GHA into various molecule tests.

Reply:
will file a new bug for this